Homeworks/module15.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+url = "https://api.punkapi.com/v2/beers/8"
+
+response_get = requests.get(url)
```

Comment:
YOu should separate it to 2 different tests functions.
I will approve it cause all main step were correct